docs/intro/tutorial.rst

```diff
@@ -114,7 +115,7 @@ As you can see, our Spider subclasses :class:`scrapy.Spider <scrapy.Spider>`
 and defines some attributes and methods:
 
 * :attr:`~scrapy.Spider.name`: identifies the Spider. It must be
-  unique within a project, that is, you can't set the same name for different
+  unique within a project, i.e., you can't set the same name for multiple
```

Comment:
This feels completely arbitrary to me.

Reply:
That's fair. The modification just made it feel more natural in my opinion.